### PR TITLE
OCD 3678: add aqa tests for SED data validation on listing upload

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: e2e/components/login/login.po.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ class LoginComponent {
   }
 
   toggleLoginComponent() {
-    $(this.elements.loginToggle).scrollAndClick();
+    $(this.elements.loginToggle).click();
   }
```
Comment (palakpatel311) [inline e2e/components/login/login.po.js:22]: There has been many failure due to changes made to this login component.
Can we please try not to change this and find workaround which may have triggered this to change?

**Review by palakpatel311 — APPROVED** [review]

File: e2e/components/login/login.po.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ class LoginComponent {
...
 
   toggleLoginComponent() {
-    $(this.elements.loginToggle).scrollAndClick();
+    $(this.elements.loginToggle).click();
```
Comment (palakpatel311) [inline e2e/components/login/login.po.js]: There has been many failure due to changes made to this login component.
Can we please try not to change this and find workaround which may have triggered this to change?

**Review by palakpatel311 — CHANGES_REQUESTED** [review]

File: e2e/components/login/login.po.js (hunk 2 above)
Comment (ashwinimore) [reply e2e/components/login/login.po.js]: I am working on it. I am unable to test locally (scrollAndClick() function fails on my local) when I reverse the change. I am going to do it anyway and test it on bamboo

Comment (ashwinimore) [reply e2e/components/login/login.po.js]: Actually no, the alternate option is needed as the tests failed on QA too..will continue to work on it

**Review by palakpatel311 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**